src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,87 @@
-export const Pagination = () => {};
+import React from "react";
+import cn from "classnames";
+import { getNumbers } from "../../utils";
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+  page: string[];
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  page,
```

Comment:
You can remove the page prop from Pagination component and count pages by using 2 params total and per page inside of it.